apps/src/utils.js

```diff
@@ -747,3 +747,11 @@ export function getTabId() {
   trySetSessionStorage('tabId', Math.random() + '');
   return tryGetSessionStorage('tabId', false);
 }
+
+export function createHiddenPrintWindow(src) {
+  var iframe = $('<iframe id="print_frame" style="display: none"></iframe>'); // Created a hidden iframe with just the desired image as its contents
+  iframe.appendTo("body");
+  iframe[0].contentWindow.document.write("<img src='" + src + "'/>");
+  iframe[0].contentWindow.document.write("<script>if (document.execCommand('print', false, null)) {  } else { window.print();  } </script>");
```

Comment:
I know you're just moving this code, but can you just negate the conditional to get rid of the empty block and the `else`?